### ISSUE TITLE
Test TfL Shield

### ISSUE THREAD

**Issue by Floppy** [theodi/shared#820]

**Comment by pikesley** [comment]
Installed [Proximo](https://devcenter.heroku.com/articles/proximo) and eventually got it working (did not want to play *at all* on a PR deploy)

And I don't think it solves our problem: it runs like

    web: bin/proximo rackup -s puma -p $PORT

So _incoming_ traffic can be aimed at that fixed IP

On essence, then, today has been a bust

**Comment by pikesley** [comment]
I have put [Quotaguard](https://devcenter.heroku.com/articles/quotaguardstatic) in the game, and I *think* I've convinced Curb to use it (after a lot of pain), but it's odd.

I did a GET, from the console, to a URL on my VPS so I could see the source IP. I did this from my laptop and from a console session on Heroku, and I saw the same IP in each case, `162.158.38.112`. And I saw the request count increasing on the Quotaguard dashboard.

However Quotaguard informs me that our fixed IPs should in fact be `52.16.121.13` & `54.154.218.66`

In short, then, I'm baffled
